service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -30,6 +39,38 @@ public Claim insertClaim(Claim claim) {
     return claim;
   }
 
+  @Override
+  public void insertAssessmentResult(
+      UUID claimId, AbdEvidenceWithSummary evidenceResponse, String diagnosticCode) {
+    ClaimEntity claimEntity = claimRepository.findById(claimId).orElse(null);
+    if (claimEntity == null) {
+      log.warn("Could not match Claim ID in insertAssessmentResult, exiting.");
+      return;
+    }
+    if (evidenceResponse.getEvidenceSummary() == null) {
+      log.warn("Evidence Summary is empty in insertAssessmentResult, exiting.");
+      return;
+    }
+    Map summary = evidenceResponse.getEvidenceSummary();
+    AssessmentResultEntity assessmentResultEntity = new AssessmentResultEntity();
+    assessmentResultEntity.setEvidenceCountSummary(summary);
+    ContentionEntity contention = findContention(claimEntity, diagnosticCode);
```

Comment:
We are not error handling here. If findContention logs an error we should not continue here. I would just return null from findContention and test that and return here.